binary_transparency/firmware/devices/usb_armory/README.md

```diff
@@ -6,25 +6,99 @@ In this package is support for using the
 transparency demo.
 
 Since the SoC on the hardware already has ROM we can't patch that to root our
-trust there, so for now we'll simply use the EL3 bootloader to "enforce" the
-correctness of the proof bundle before chaining to a kernel. However, given the
-rich set of secure hardware on board, this could be changed to leverage that as
-the root of trust.
+trust there, so for now we'll simply use a first-stage EL3 bootloader to
+"enforce" the correctness of the proof bundle before chaining to something
+(e.g. a full linux image based app) that represents the firmware being made
+discoverable.
+
+The enforcement code not being in the masked ROM is an obvious shortcoming of this
+demo, however given the interesting array of security hardware on this board
+it should be possible to use some of that as an alternative trust base.
 
 Storage
 -------
 
 > :warning: these are scratch notes, not yet reflective of reality, and so may
 > change drastically!
 
-The _plan_ is to partition the device SD card into 3:
-1. `Boot` partition, this will contain our "fake ROM" which does the proof verification
-3. `Bundle` partition, this will contain the proof bundle
-4. `Firmware` partition, this contains the logged firmware image to boot if
-    verification succeeds.
+We'll use the µSD card slot of the USB Armory for our purposes.
+
+The SD card will contain:
+   - our "enforcing" [bootloader](./bootloader)
+   - the firmware being made discoverable
```

Comment:
"discoverable" terminology is somewhat new. We should introduce the terminology here (with a link to https://github.com/google/trillian/blob/master/docs/claimantmodel/Logs.md), or use different terminology.

Reply:
-> transparent for now